### PR TITLE
Update: icons and contributors

### PR DESCRIPTION
**Pull request by Juancxh**
Icons: Added some icons to improve README
Contributors: Added a feature recognizing contributors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,55 @@
-<p align="center">
-  <img src="public/assets/img/logo-without-bg.png" width="320"/>
-</p>
-
+![OpenTech](https://i.ibb.co/4g2hgty/landing.png)
```
Comment (paulotijero) [inline README.md:1]: This logo is different from the original, please return the old one. This logo looks weird:
Screenshot from 2020-09-02 18-10-41

File: README.md [hunk 2]
```diff
@@ -1,34 +1,55 @@
...
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-1-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
 [![Netlify Status](https://api.netlify.com/api/v1/badges/f6ad4d4e-3128-43db-a06e-6f63790cba6c/deploy-status)](https://app.netlify.com/sites/opentech/deploys) [![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/opentech-pe/opentech.pe/blob/master/LICENSE) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](http://makeapullrequest.com)
```
Comment (paulotijero) [inline README.md:5]: Add contributors badged inside the badges block
```suggestion
[![Netlify Status](https://api.netlify.com/api/v1/badges/f6ad4d4e-3128-43db-a06e-6f63790cba6c/deploy-status)](https://app.netlify.com/sites/opentech/deploys) [![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/opentech-pe/opentech.pe/blob/master/LICENSE) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](http://makeapullrequest.com)[![All Contributors](https://img.shields.io/badge/all_contributors-1-orange.svg?style=flat-square)](#contributors-)
```

File: README.md [hunk 3]
```diff
@@ -1,34 +1,55 @@
-<p align="center">
-  <img src="public/assets/img/logo-without-bg.png" width="320"/>
-</p>
-
+![OpenTech](https://i.ibb.co/4g2hgty/landing.png)
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-1-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
```
Comment (paulotijero) [inline README.md:4]: I think this block is unnecessary

**Review by paulotijero — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Juancxh) [reply README.md:5]: Si se quiere actualizar el número de contribuidores automaticamente es necesario lo siguiente:

 
https://allcontributors.org/docs/en/bot/faq

File: README.md (hunk 1 above)
Comment (Juancxh) [reply README.md:1]: Para que el badge no pierda el formato con el:
```
<p>
</p>
```
Se optó por la opción presentada, pero entiendo lo del logo. Se corregirá inmediatamente

**Comment by Juancxh** [comment]
All changes in the following link:
https://github.com/opentech-pe/opentech.pe/commit/86c4f495e30422ad5a72ac9c360b990216873b5b

**Pull request closed without merging**